python/foglamp/plugins/south/mqtt_sparkplug/mqtt_sparkplug.py

```diff
@@ -37,37 +37,43 @@
         'description': 'Name of Asset',
         'type': 'string',
         'default': 'mqtt',
-        'order': '1'
+        'order': '1',
+        'displayName': 'Asset Name'
     },
     'url': {
         'description': 'URL for MQTT Server',
         'type': 'string',
         'default': 'chariot.groov.com',
-        'order': '2'
+        'order': '2',
+        'displayName': 'URL'
```

Comment:
as we have port as another key, it should have been been host, atleast we should say so in displayName `MQTT Host`